src/modules/api/user/link/link.controller.ts

```diff
@@ -1,7 +1,21 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get, Param } from '@nestjs/common';
 import { LinkService } from './link.service';
 
 @Controller('users/:userId/links')
 export class LinkController {
-  constructor(private linkService: LinkService) {}
+  constructor(private readonly linkService: LinkService) {}
+  @Get()
+  async getAllLinksForUserId(@Param('userId') userId:string) {
+    const links = await this.linkService.getAllLinksForUser(userId);
+    if (links.length) {
+      return {
+        status: 'OK',
+        result: JSON.stringify(links),
```

Comment:
You wouldn't need stringify. Increases burden on front-end to parse it. Just send as-is